tests/unit/app/models/test_models.py

```diff
@@ -94,38 +93,6 @@ def test_llm_response():
         assert llm_response.referenced_documents == referenced_documents
         assert not llm_response.truncated
 
-    @staticmethod
-    def test_llm_response_referenced_documents_as_json():
-        """Test the LLMResponse model when referenced documents are passed as string with JSON."""
-        conversation_id = "id"
-        response = "response"
-
-        # check the object hook defined for ReferenceDocument
-        # data class
-        referenced_documents = json.loads(
-            """
-                [
-                    {
-                     \"docs_url\": \"https://foo.bar.com/index.html\",
-                     \"title\": \"Foo Bar\"
-                    }
-                ]
-                """,
-            object_hook=ReferencedDocument.json_decode_object_hook,
-        )
-
-        llm_response = LLMResponse(
-            conversation_id=conversation_id,
-            response=response,
-            referenced_documents=referenced_documents,
-            truncated=False,
-        )
-
-        assert llm_response.conversation_id == conversation_id
-        assert llm_response.response == response
-        assert llm_response.referenced_documents == referenced_documents
-        assert not llm_response.truncated
```

Comment:
The removed method was used only in tests.
This test was testing that reference doc can be loaded from json.
We can do something like this for every model (dataclass), but I don't see value in doing this.
```python
def test_referenced_docs_from_json():
    """Test the ReferencedDocument model can be loaded from JSON."""
    ref_docs = ReferencedDocument("a", "b")
    ref_docs_json = json.dumps(ref_docs.__dict__)
    assert ReferencedDocument(**json.loads(ref_docs_json)) == ref_docs
```